source/documentation/admin-guide/appe-oVirt_and_SSL.html.md

```diff
@@ -92,15 +92,27 @@ The internal CA stores the internally generated key and certificate in a **P12**
         SSL_CERTIFICATE=/etc/pki/ovirt-engine/certs/apache.cer
         SSL_KEY=/etc/pki/ovirt-engine/keys/apache.key.nopass
 
-10. Restart the `ovirt-provider-ovn` service:
+10. Edit the **/etc/ovirt-imageio-proxy/ovirt-imageio-proxy.conf** 
+        
+        # vi /etc/ovirt-engine/ovirt-websocket-proxy.conf.d/10-setup.conf
+
+   Make the following changes and save the file:
+
+        # Key file for SSL connections
+        ssl_key_file = /etc/pki/ovirt-engine/keys/apache.key.nopass
+
+        # Certificate file for SSL connections
+        ssl_cert_file = /etc/pki/ovirt-engine/certs/apache.cer
+
+11. Restart the `ovirt-provider-ovn` service:
 
         # systemctl restart ovirt-provider-ovn.service
         
-11. Restart the `ovirt-websocket-proxy` service:
+12. Restart the `ovirt-websocket-proxy` service:
```

Comment:
We also need to restart ovirt-imageio-proxy service to reload the new configuration.